spec/components/candidate_interface/work_history_review_component_spec.rb

```diff
@@ -150,27 +150,10 @@
       it 'renders component with consolidated work breaks and without individual breaks' do
         result = render(explanation: 'I was sleeping.')
 
-        expect(result.css('.app-summary-card__title').text).not_to include('You have a break in your work history in the last 5 years')
         expect(result.css('.govuk-summary-list__key').text).to include(t('application_form.work_history.breaks.review_label'))
         expect(result.css('.govuk-summary-list__actions').text).to include(t('application_form.work_history.breaks.change_action'))
       end
     end
-
-    context 'when individual breaks are editable' do
-      it 'renders component without break placeholders' do
-        result = render(editable: true)
-
-        expect(result.css('.app-summary-card__title').text).to include('You have a break in your work history in the last 5 years')
-      end
-    end
```

Comment:
I think we need to add two tests here:

1. one that checks that the placeholder renders when there is a break between two jobs
2. one that checks that a placeholder doesn't render if they're currently in a job role (i.e. end date is nil on their latest job)